lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -641,6 +641,12 @@ def _from_file(self, root_path: Path, file_path: Path) -> List[dict]:
                 logger.error("Failed to read alert rules from %s: %s", file_path.name, e)
                 return []
 
+            if not rule_file:
+                logger.warning("Empty rule file: %s", file_path.name)
```

Comment:
just to be consistent with the error message in another leaf
```suggestion
                logger.warning("Empty rules file: %s", file_path.name)
```

PS
I would even prefer initial implementation with a single check :)
but do not have a strong opinion on this